From dc1c8b9612bb78c2df5e789fbdfe75515b06accf Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Mon, 5 Feb 2018 18:23:27 +0800
Subject: [PATCH 3503/5242] MLK-17230-11: dts: disable mjpeg decoder and
 encoder

commit  d46284321b4702a8885da137007662d11541d17a from
https://source.codeaurora.org/external/imx/linux-imx.git

Disable mjpeg decoder and encoder.

Reviewed-by: Sandor.Yu <sandor.yu@nxp.com>
Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
(cherry picked from commit 8aac8c4e9cb1778c057ce4170b7c1395aaaf6f1d)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qxp-mek-csi.dts     |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-csi.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-csi.dts
index 9a98474..110c29c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-csi.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-csi.dts
@@ -39,6 +39,14 @@
 	};
 };
 
+&jpegdec {
+	status = "disabled";
+};
+
+&jpegenc {
+	status = "disabled";
+};
+
 &i2c0_csi0 {
 	status = "disabled";
 };
-- 
1.7.9.5

